src/Components/PresentationalComponents/CVEInfoBox/CVEInfoBox.scss

```diff
@@ -36,5 +36,5 @@
 }
 
 .severity-color-important {
-    background-color: #ec7a08 !important;
+    background-color: var(--pf-global--palette--orange-300) !important;
```

Comment:
Intentional, doesn't change functionality.

Reply:
great! it's always better to use the variables